h2o-core/src/main/java/hex/ModelMetrics.java

```diff
@@ -127,19 +127,25 @@ static public double getMetricFromModel(Key<Model> key, String criterion) {
   }
 
   static public double getMetricFromModelMetric(ModelMetrics mm, String criterion) {
-    if (null == criterion || criterion.equals("")) throw new H2OIllegalArgumentException("Need a valid criterion, but got '" + criterion + "'.");
+    if (null == criterion || criterion.equals(""))
+      throw new H2OIllegalArgumentException("Need a valid criterion, but got '" + criterion + "'.");
+
     Method method = null;
+    Object obj = null;
+    criterion = criterion.toLowerCase();
     ConfusionMatrix cm = mm.cm();
     try {
-      method = mm.getClass().getMethod(criterion.toLowerCase());
+      method = mm.getClass().getMethod(criterion);
+      obj = mm;
     }
     catch (Exception e) {
       // fall through
```

Comment:
I am aware that this is outside your PR, just noting: here the exception is _ignored_, perhaps it's my poor English but _fall through_ gives me completely different intuition for handling than what happens here. 
Besides, such frequent exception swallowing, even in obvious cases like this, is a little bit disturbing. I guess that in most these cases, at least `IllegalStateException` could have been thrown.

Reply:
the execution of this method through to the next step ;)

an exception is eventually thrown if neither of the attempts succeeds